Automattic/jetpack
# Premium Content Block: Add missed blockprops change

Follow up to https://github.com/Automattic/jetpack/pull/19844

#### Changes proposed in this Pull Request:

* Replace the experimental block with blockProps.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Insert a premium content block in a post, and confirm it looks and works correctly.
* Use an existing post with a premium content block, and confirm it continues to work correctly.
* Publish a post with the block and confirm it works on the front end.
